Mathlib/Data/Matrix/Basis.lean

```diff
@@ -416,6 +416,21 @@ theorem mem_range_scalar_iff_commute_single' {M : Matrix n n α} :
 @[deprecated (since := "2025-05-05")]
 alias mem_range_scalar_iff_commute_stdBasisMatrix' := mem_range_scalar_iff_commute_single'
 
+/-- The center of `Matrix n n α` is equal to the image of the center of `α` under `scalar n`. -/
+theorem center_eq_scalar_image :
+    Set.center (Matrix n n α) = scalar n '' Set.center α := Set.ext fun x ↦ by
```

Comment:
Should we use `Submonoid.center` instead?

Reply:
In theory you could repeat this lemma for every bundled version, replacing `''` with `.map`

Reply:
But we should definitely keep the current one either way.